### PR TITLE
chore(conf) Improve KIND makefile

### PR DESCRIPTION
**Pull request by nickolaev**
### Summary

Split the KIND specific parts into a separate Makefile.kind.mk. Update the KIND version to 0.8.0 and
the installed K8s to 1.15.11. Add `stop/kind` make target to destroy the created KIND cluster.

### Full changelog

* chore(conf) improve kind make targets
* chore(conf) separate kind specifics to Makefile.kind.mk
* chore(*) bump k8s image version in kind to 1.15.11
* chore(*) bump kind version to 0.8.0
* chore(*) add stop/kind make target

### Issues resolved

None

### CODE REVIEW

File: DEVELOPER.md [hunk 1]
```diff
@@ -225,7 +225,7 @@ export KUBECONFIG="$(kind get kubeconfig-path --name=kuma)"
 2. Run `Control Plane` on local machine:
 
 ```bash
-make run/k8s
+make kind/deploy/kuma
```
Comment (jakubdyszkiewicz) [inline DEVELOPER.md:228]: this should still be `run/k8s`. This section of instruction is to run Kuma locally and connect to K8S in Kind.

File: DEVELOPER.md [hunk 2]
```diff
@@ -260,13 +260,13 @@ export KUBECONFIG="$(kind get kubeconfig-path --name=kuma)"
 2. Deploy `Control Plane` to [KIND](https://kind.sigs.k8s.io/docs/user/quick-start) (Kubernetes IN Docker):
 
 ```bash
-make start/control-plane/k8s
+make kind/deploy/kuma
```
Comment (jakubdyszkiewicz) [inline DEVELOPER.md:263]: does this instruction work like described? or should I also run 
```
make images && make docker/save
```
?

Comment (nickolaev) [reply DEVELOPER.md:263]: `kind/deploy/kuma` depends on `kind/load`, which in turn depends on `kind/load/control-plane` etc. Then al these targets depend on `image/kuma-cp` etc., which then builds the app and the docker images. The trick is that kind actually loads the images straight from the local docker registry and does not need the intermediate step of saving the images. So yep, that is all we need here. And if not, I will fix it.

Comment (jakubdyszkiewicz) [reply DEVELOPER.md:263]: That's great! I forgot KIND uses the local docker registry.

**Pull request merged**